components/director/pkg/formationconstraint/formation_constraint_input.go

```diff
@@ -57,8 +60,11 @@ type ConfigMutatorInput struct {
 	Operation                              model.FormationOperation `json:"operation"`
 	ResourceType                           model.ResourceType       `json:"resource_type"`
 	ResourceSubtype                        string                   `json:"resource_subtype"`
+	NotificationStatusReportAddress        uintptr                  `json:"notification_status_report_address"`
```

Comment:
`NotificationStatusReportMemoryAddress`? to be consistent with other fields
P.S. Valid for the same field in the destination creator input